Packs/ExpanseV2/ReleaseNotes/1_9_5.md

```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Cortex Xpanse
+- **Breaking Changes:** The `expanse-get-risky-flows` and `expanse-list-risk-rules` commands are no longer supported in the Xpanse API, and have been deprecated.
```

Comment:
```suggestion
- **Breaking Changes:** The ***expanse-get-risky-flows*** and ***expanse-list-risk-rules*** commands are no longer supported in the Xpanse API, and have been deprecated.
```

Reply:
let's use the [`-bc` flag](https://xsoar.pan.dev/docs/documentation/release-notes#breaking-changes-version)